### ISSUE TITLE
Style guide needs to be enforced

### ISSUE THREAD

**Issue by KrishnaPravin** [covid19india/covid19india.github.io#154]
**Describe the bug**
Currently, there is no guidance/control of the style of the code here. As a result, in every PR there are lots of changes related to spaces, newline characters, and other code style-related changes. This increases the number of changes.

**To Reproduce**
1. Check the number of changes related to whitespace in [PRs](https://github.com/covid19india/covid19india-react/pull/92/files#diff-12e36ec8e5ae32b59f06724c9ed7f370)

**Expected behavior**
The contributors should run a code formatting tool like [Prettier](https://prettier.io/) before submitting the PR.
To help the contributors do this, there must be a config file that has the formatting rules in it. In the case of Prettier, it is `.prettierrc` at the root of the project or any other [configuration format supported by Prettier](https://prettier.io/docs/en/configuration.html).

**Screenshots**
<img width="1067" alt="Screenshot 2020-03-26 at 11 52 49 AM" src="https://user-images.githubusercontent.com/17372422/77616728-5a3d6680-6f58-11ea-951a-6db67a1ea0c1.png">

**Comment by abhishekakade** [comment]
There are also CRLF and LF issues. Decide which one to go with. ESLint throws errors/warnings because by default it expects line breaks to be LF while they are CRLF.
`Expected linebreaks to be 'LF' but found 'CRLF'.eslint(linebreak-style)`

**Comment by satishsingh35** [comment]
Can i pick this up.? Can some one help me with the process of claiming a issue here

**Comment by KrishnaPravin** [comment]
> Can i pick this up.? Can some one help me with is the process of claiming a issue here

@jeremyphilemon  @JunaidBabu  please guide

**Comment by abhishekakade** [comment]
@jeremyphilemon @JunaidBabu @satishsingh35 sorry, I worked on the issue without getting it assigned to me first. There was power outage since morning and I just checked the `.eslintrc.js` file and since it was unchanged, decided to make a pr now.

**Comment by vinay-thallam** [comment]
@KrishnaPravin  @abhishekakade @jeremyphilemon  @JunaidBabu 

This issue is duplicate of [https://github.com/covid19india/covid19india-react/issues/162](url).

@abhishekakade  I think we worked for this in parallel without knowing.

Please check the below PR. 
https://github.com/covid19india/covid19india-react/pull/212
Some additional things apart from prettier setup were done in this. Please check and let me know your opinion.

**Comment by KrishnaPravin** [comment]
duplicate #162
